CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+
+## [2.62.0](https://github.com/openfoodfacts/openfoodfacts-server/compare/v2.61.0...v2.62.0) (2025-03-05)
```

Comment:
AFAIK, the Changelog file should only be modified by the automated release process?

Reply:
yes @Kout95 it's Realease please that handle the changelog.